views/js/qtiCreator/widgets/interactions/states/Question.js

```diff
@@ -23,10 +24,18 @@ define([
 
 
     }, function(){
-
         //destroy and hide it
         this.widget.$form.empty().hide();
-
+        //render textReaderInteraction possible xi:include elements
+        let element = this.widget.element;
+        const baseUrl = this.widget.options.baseUrl;
+        if (element.properties && Array.isArray(element.properties.pages) && element.typeIdentifier === 'textReaderInteraction') {
+            const pagePromises = xincludeLoader.loadByElementPages(element.properties.pages, baseUrl)
+            Promise.all(pagePromises).then(updatedPages => {
+                element.properties.pages = updatedPages;
+                element.widgetRenderer.renderPages(element.properties);
+            });
+        }
```

Comment:
**quibble:** 😱 this is specific to a particular interaction, which is a PCI... it must be added to the Question state of the PCI, not to the state that any interaction will use... Please avoid spaghetti code.